go.mod

```diff
@@ -53,7 +53,6 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/google/go-github/v37 v37.0.0
 	github.com/google/uuid v1.6.0
-	github.com/goreleaser/goreleaser v1.1.0
```

Comment:
I assume those changes are from `go mod tidy`? If so those should be fine.

Reply:
that's right